Basic_viewer/doc/Basic_viewer/CGAL/Qt/Basic_viewer_qt.h

```diff
@@ -0,0 +1,256 @@
+namespace CGAL {
+
+//------------------------------------------------------------------------------
+/*!
+ \ingroup PkgBasicViewerClasses
+
+The class `Basic_viewer_qt` is a Qt widget based on `QGLViewer` that allows to visualize 3D elements: points, segments, triangles, rays and lines. This class stores a reference to a `Graphics_scene`. Elements can either be added directly in the viewer or through the scene. This class requires `CGAL_Qt5`, and is only available if the macro `CGAL_USE_BASIC_VIEWER` is defined. Linking with the cmake target `CGAL::CGAL_Basic_viewer` will link with `CGAL_Qt5` and add the definition `CGAL_USE_BASIC_VIEWER`.
+
+\tparam BufferType the number type used for point coordinates: `float` by default.
```

Comment:
Why is it not `NumberType`. Also can it be anything else than `double` or `float`? Maybe `Precision` ?